docs/decisions/0004-event-bus-as-extension-of-django-signals.rst

```diff
@@ -0,0 +1,18 @@
+1 Event bus as extension of django signals
+------------------------------------------
+
+1.1 Context
+~~~~~~~~~~~
+
+- Django Signals are widely used in edx-platform and elsewhere. OpenedX developers are familiar with Django Signals.
+
+- Django Signals are much cheaper to send, and offer pretty much no additional work as compared to sending via Kafka.
+
+- Django Signals don’t require extra infrastructure and are simpler for people to test with.
+
+- We want to make it easy to integrate different event bus technologies
+
+1.2 Decision
+~~~~~~~~~~~~
+
+Django signals will be used as the event bus for internal service communication and external event bus will be implemented as an app listening for relevant Django signals and passing it on to external event bus.
```

Comment:
I think we want to be more explicit about what the behavior will be. Django signals will be used communicate messages within a service. Django signals will also be passed onto the event bus, and then when a service receives a message from the event bus, it will then emit a Django signal?